ixofoundation/ixo-world
# chore(fix): start now button and video caption text (#62)

## Scope 
Fix start now button

## Work done
Add back start now button and update video component text

## Steps to test
See 'Start Now' button and updated video text that reads 'Get your mobile Impact Wallet'

## Screenshot
<img width="1157" alt="Screenshot 2020-05-07 at 10 41 21" src="https://user-images.githubusercontent.com/35291291/81273536-4d2f8f00-904f-11ea-83ec-9cc59e8ee950.png">
